test_utilities/src/utilities.rs

```diff
@@ -1161,24 +1169,12 @@ pub fn test_db(
     let mut environment_delete = environment.clone();
     environment_delete.action = Action::Delete;
     let ea = EnvironmentAction::Environment(environment.clone());
-    let ea_delete = EnvironmentAction::Environment(environment_delete);
+    let ea_delete = EnvironmentAction::Environment(environment_delete.clone());
 
-    match provider_kind {
-        Kind::Aws => match aws_deploy(&context, ea) {
-            TransactionResult::Ok => assert!(true),
-            TransactionResult::Rollback(_) => assert!(false),
-            TransactionResult::UnrecoverableError(_, _) => assert!(false),
-        },
-        Kind::Do => match do_deploy(&context, ea, DO_TEST_REGION) {
-            TransactionResult::Ok => assert!(true),
-            TransactionResult::Rollback(_) => assert!(false),
-            TransactionResult::UnrecoverableError(_, _) => assert!(false),
-        },
-        Kind::Scw => match scw_deploy(&context, ea, SCW_TEST_ZONE) {
-            TransactionResult::Ok => assert!(true),
-            TransactionResult::Rollback(_) => assert!(false),
-            TransactionResult::UnrecoverableError(_, _) => assert!(false),
-        },
+    match environment.deploy_environment(provider_kind.clone(), &context, &ea) {
```

Comment:
`environment.deploy_environment` sounds weird, there is a naming issue or fuzzy trait definition here,
if environment to be kept, then consifer rename the function to `deploy` so it's smoother => `environment.deploy(`

Reply:
it could be environement deploy_charts. That's why I kept it specific